cfme/scripting/quickstart/system.py

```diff
@@ -149,13 +149,16 @@ def retry_install_once(command, shell):
         run_cmd_or_exit(command, shell=shell)
 
 
-def install_system_packages():
+def install_system_packages(debuginfo_install=False):
+    installed = False
     if INSTALL_COMMAND and REQUIRED_PACKAGES:
         retry_install_once(INSTALL_COMMAND + REQUIRED_PACKAGES, shell=True)
-    if DEBUG_INSTALL_COMMAND and DEBUG_PACKAGES:
+        installed = True
+    if DEBUG_INSTALL_COMMAND and DEBUG_PACKAGES and debuginfo_install:
```

Comment:
I believe we need to pass the buck to ansible in such scripts. we reinvent the wheel

Reply:
but we make the _best_wheels! :)  I don't disagree at all, the system package mgmt is the biggest headache of quickstart overall. Maybe we keep the components that are determining OS and install ansible, and then run the playbook for the given OS, instead of trying to build bash commands to run.